frontend/templates/lobby.html

```diff
@@ -1610,6 +1610,31 @@
 
             return circleData;
         }
+        
+        let sizeSlot = 8;
+        function wheelSize(e) {
+            event.preventDefault();
+            sizeSlot += Math.min(Math.max(-1, e.deltaY), 1);
```

Comment:
Why is this necessary if the sizes are fixed numbers?

Reply:
sizes are not fixed numbers. 🤔
see https://bekeshbekesh.herokuapp.com/

Reply:
i don't really understand ^^

There's 4 base sizes that have fixed numbers: 8,16,24 and 32

What do you mean?

Reply:
![image](https://user-images.githubusercontent.com/46781518/118123803-031f7d00-b40a-11eb-8a7c-774b983b2945.png)
in this feature you can choose something between 8 and 16 ... like 10
but it marks closest brush size on the bottom =) 
you can see the feature on my fork version.

Reply:
I see, I missunderstood at first. However, how would people with touchscreen achieve the same sizes then?
I'd like to keep features across devices mostly the same. Do you have any idea?

Reply:
it's not that important I think
but a text box with "brush size" title? =) 
it's not a good thing I guess
so, nevermind. close the pool request

Reply:
Generally I don't think it's a bad idea, as I also just added shortcuts for switching. I also want to add a better colorpicker on the canvas itself.

Now you wouldn't be able to set these inbetween sizes on mobile. It's also a bit weird that we'd display one of the radio buttons as checked, even though it's not technically true.

Sorry, just a bit perfectionist here :D

Reply:
it's ok =) so good luck